tests/test_utils.py

```diff
@@ -248,10 +248,10 @@ async def decorated_always_fail(*args, **kwargs):
 
 # create_temp_creds {{{1
 def test_create_temp_creds():
-    with mock.patch.object(utils, "createTemporaryCredentials") as p:
-        p.return_value = {"one": b"one", "two": "two"}
+    with mock.patch.object(utils, "createTemporaryCredentials") as patched:
```

Comment:
renamed `p` to `patched` as it conflicts with pdb `p` command; took me longer than I'd like to admit to realize this is why I had issues debugging it 🥲